src/lib/get-link.ts

```diff
@@ -0,0 +1,63 @@
+import { SupportedLocale } from 'common/lib/locale';
+
+// English is always required, and the other locales are optional.
+type TranslatedLink = Partial< Record< SupportedLocale, string > > & { en: string };
+
+const DOCS_LINKS = {
+	docsStudio: {
+		en: 'https://developer.wordpress.com/docs/developer-tools/studio/',
+		es: 'https://developer.wordpress.com/es/docs/herramientas-para-desarrolladores/studio/',
+	},
+	docsImportExport: {
+		en: 'https://developer.wordpress.com/docs/developer-tools/studio/import-export/',
+		es: 'https://developer.wordpress.com/es/docs/herramientas-para-desarrolladores/studio/importar-exportar/',
+	},
+	docsSites: {
+		en: 'https://developer.wordpress.com/docs/developer-tools/studio/sites/',
+		es: 'https://developer.wordpress.com/es/docs/herramientas-para-desarrolladores/studio/sitios/',
+	},
+	docsSync: {
+		en: 'https://developer.wordpress.com/docs/developer-tools/studio/sync/',
+		es: 'https://developer.wordpress.com/es/docs/herramientas-para-desarrolladores/studio/sync/',
+	},
+	docsCli: {
+		en: 'https://developer.wordpress.com/docs/developer-tools/studio/cli/',
+	},
+} satisfies Record< `docs${ string }`, TranslatedLink >;
+
+const BLOG_LINKS = {
+	blogPhpVersions: {
+		en: 'https://wordpress.com/blog/2025/03/17/studio-wordpress-php-versions/',
+		es: 'https://wordpress.com/es/blog/2025/04/02/modifica-las-versiones-de-wordpress-y-php-de-tu-sitio-local-con-studio/',
+		fr: 'https://wordpress.com/fr/blog/2025/03/28/studio-wordpress-php-versions/',
+		'pt-br': 'https://wordpress.com/pt-br/blog/2025/04/02/estudio-wordpress-php-versoes/',
+	},
+	blogPreviewSites: {
+		en: 'https://wordpress.com/blog/2025/02/24/studio-preview-sites/',
+		es: 'https://wordpress.com/es/blog/2025/03/05/colabora-mas-y-mejor-en-studio-con-los-sitios-de-vista-previa/',
+		fr: 'https://wordpress.com/fr/blog/2025/03/04/sites-de-previsualisation-studio/',
+		ja: 'https://wordpress.com/ja/blog/2025/03/28/studio-preview-sites/',
+		'pt-br': 'https://wordpress.com/pt-br/blog/2025/03/06/estudio-visualizar-sites/',
+	},
+	blogCustomDomainsHttps: {
+		en: 'https://wordpress.com/blog/2025/03/31/studio-custom-domains-https/',
+		es: 'https://wordpress.com/es/blog/2025/03/31/studio-custom-domains-https/',
+		'pt-br': 'https://wordpress.com/pt-br/blog/2025/04/03/estudio-dominios-personalizados-https/',
+	},
+} satisfies Record< `blog${ string }`, TranslatedLink >;
```

Comment:
These types force the link keys to start with `blog` and `docs`, respectively.

Reply:
One last nit on method signature - if we want to enforce a prefix, would it be clearer to change the method signature and add another parameter for that?

```
 getLink( 'en', 'docs', 'importExport' )
 ```

Alternatively, what if we skipped enforcing that, and used one array for both docs and blog links, making it generic and not tied to the resource type it stores?

```
const LINKS = {
	blogPhpVersions: {
		[...]
	},
	docsPhpVersions: {
		[...]
	},
	dotOrgBlogGutenber: {
		[...]
	},
}

Reply:
I prefer having everything in the same object. I'm not sure if forcing a prefix is useful, but I see that it can help with decision-making when seeing the autocomplete while calling the function.

![Screenshot 2025-05-13 at 09 49 07](https://github.com/user-attachments/assets/55d44622-7eb3-499e-a86d-c34bfea59d55)

I'll leave it as it is for now. We can always merge the objects and remove the prefix check later.

Reply:
Sounds good, thanks for sharing this use case.